corehq/apps/smsbillables/management/commands/pricing_data/twilio-rates-2015_10_06.csv

```diff
@@ -0,0 +1 @@
UG,Uganda,UGANDA Outbound SMS - UT Mobile,0.03
UG,Uganda,UGANDA Outbound SMS - Warid,0.065
US,United States,UNITED STATES Outbound SMS - Other,0.0075
UY,Uruguay,URUGUAY Outbound SMS - Antel,0.04243
UY,Uruguay,URUGUAY Outbound SMS - Claro,0.02817
UY,Uruguay,URUGUAY Outbound SMS - Other,0.02817
UY,Uruguay,URUGUAY Outbound SMS - movistar,0.02817
UZ,Uzbekistan,UZBEKISTAN Outbound SMS - Beeline,0.013
UZ,Uzbekistan,UZBEKISTAN Outbound SMS - Other,0.013
UZ,Uzbekistan,UZBEKISTAN Outbound SMS - Perfectum,0.06
UZ,Uzbekistan,UZBEKISTAN Outbound SMS - Ucell,0.02817
UZ,Uzbekistan,UZBEKISTAN Outbound SMS - Uzdunrobita,0.09
VC,St Vincent Grenadines,ST. VINCENT & GRENADINES Outbound SMS - Digicel,0.04983
VC,St Vincent Grenadines,ST. VINCENT & GRENADINES Outbound SMS - LIME,0.02
VC,St Vincent Grenadines,ST. VINCENT & GRENADINES Outbound SMS - Other,0.02
VE,Venezuela,VENEZUELA Outbound SMS - Digitel,0.026
VE,Venezuela,VENEZUELA Outbound SMS - Movilnet,0.04
VE,Venezuela,VENEZUELA Outbound SMS - Other,0.026
VE,Venezuela,VENEZUELA Outbound SMS - movistar,0.04
VG,"Virgin Islands, British",BRITISH VIRGIN ISLANDS Outbound SMS - CCT,0.05417
VG,"Virgin Islands, British",BRITISH VIRGIN ISLANDS Outbound SMS - Digicel,0.03
VG,"Virgin Islands, British",BRITISH VIRGIN ISLANDS Outbound SMS - LIME,0.03467
VG,"Virgin Islands, British",BRITISH VIRGIN ISLANDS Outbound SMS - Other,0.03
VI,"Virgin Islands, U.S.",US VIRGIN ISLANDS Outbound SMS,0.02
VN,Vietnam,VIETNAM Outbound SMS - Beeline,0.02
VN,Vietnam,VIETNAM Outbound SMS - EVNTelecom,0.104
VN,Vietnam,VIETNAM Outbound SMS - MobiFone,0.02
VN,Vietnam,VIETNAM Outbound SMS - Other,0.02
VN,Vietnam,VIETNAM Outbound SMS - S-Fone,0.01
VN,Vietnam,VIETNAM Outbound SMS - Vietnamobile,0.0975
VN,Vietnam,VIETNAM Outbound SMS - Viettel Mobile,0.07583
VN,Vietnam,VIETNAM Outbound SMS - Vinaphone,0.04983
VU,Vanuatu,VANUATU Outbound SMS - Digicel Vanuatu,0.04983
VU,Vanuatu,VANUATU Outbound SMS - Other,0.04983
VU,Vanuatu,VANUATU Outbound SMS - TVL,0.07
WS,Samoa,SAMOA Outbound SMS - Digicel,0.04983
WS,Samoa,SAMOA Outbound SMS - Other,0.04983
YE,Yemen,YEMEN Outbound SMS - HiTS-UNITEL,0.03
YE,Yemen,YEMEN Outbound SMS - Other,0.03
YE,Yemen,YEMEN Outbound SMS - Sabafon,0.03
YE,Yemen,YEMEN Outbound SMS - Spacetel Yemen,0.03
YE,Yemen,YEMEN Outbound SMS - Yemen Mobile,0.09
ZA,South Africa,SOUTH AFRICA Outbound SMS - 8ta,0.02167
ZA,South Africa,SOUTH AFRICA Outbound SMS - Cell C,0.02167
ZA,South Africa,SOUTH AFRICA Outbound SMS - MTN,0.02167
ZA,South Africa,SOUTH AFRICA Outbound SMS - Other,0.02167
ZA,South Africa,SOUTH AFRICA Outbound SMS - Vodacom,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - Airtel,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - MTN,0.04767
ZM,Zambia,ZAMBIA Outbound SMS - Other,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - ZAMTEL,0.04
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Econet Wireless,0.11483
ZW,Zimbabwe,ZIMBABWE Outbound SMS - NetOne,0.013
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Other,0.013
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Telecel,0.0195

```

Comment:
I was able to run the migration locally without running into problems from the unicode